src/components/atoms/ImageComponent/Image.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../styles/variables.scss';
+
+.atom__image {
+    &__medium {
+        width: $image-width-medium;
+        height: auto;
+    }
+    &large {
+        width: $image-width-large;
```

Comment:
Its enough to add this property in the base img styles, you dont need to re declare that dot every class